src/ripple/protocol/impl/Rules.cpp

```diff
@@ -83,6 +82,18 @@ bool
 Rules::enabled(uint256 const& feature) const
 {
     assert(impl_);
+
+    // The functionality of the "NonFungibleTokensV1_1" amendment is
+    // precisely the functionality of the following three amendments
+    // so if their status is ever queried individually, we inject an
+    // extra check here to simplify the checking elsewhere.
+    if (feature == featureNonFungibleTokensV1 ||
+        feature == fixNFTokenNegOffer || feature == fixNFTokenDirV1)
+    {
+        if (impl_->enabled(featureNonFungibleTokensV1_1))
+            return true;
+    }
+
```

Comment:
The only downside to doing it this way is that this check will need to be made on every call to `enabled`. It's a tiny downside, since it's a simple check. *But* what do you think of, instead of this check, you modify `Change::applyAmendment`. If the enabled feature is `featureNonFungibleTokensV1_1`, you add the other three to the amendments list, too?
https://github.com/XRPLF/rippled/blob/83aa1c1818e2854e98eebce040b9dc6c72de640e/src/ripple/app/tx/impl/Change.cpp#L195-L197

Reply:
I thought about this (and prepared a branch to do just that, actually) but a couple of people that I floated that idea by felt that it was weird, because it could result in an amendment that doesn't, on its own, have majority appearing to get arbitrarily enabled.

I do think that's a better option in the long term and will likely introduce a PR to make that possible, but I don't think it's the right option here.